docs/reference-guides/interactivity-api/api-reference.md

```diff
@@ -354,6 +359,10 @@ store( "myPlugin", {
 
 The callback passed as the reference receives [the event](https://developer.mozilla.org/en-US/docs/Web/API/Event) (`event`), and the returned value by this callback is ignored. When the element is removed from the DOM, the event listener is also removed.
 
+### `wp-on-window-async`
+
+Similar to `wp-on-async`. An optimized version of `wp-on-window` that prevents blocking the main thread on long tasks. Use this directive for any `wp-on-window` that doesn't need the `event` object (`event.preventDefault()`, `event.stopPropagation()`, etc.)
```

Comment:
```suggestion
Similar to `wp-on-async`. An optimized version of `wp-on-window` that prevents blocking the main thread on long tasks. Use this directive for any `wp-on` that doesn't need synchronous access to the `event` object, in particular the methods `event.preventDefault()`, `event.stopPropagation()`, and `event.stopImmediatePropagation()`. This event listener is also added as `passive`.
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/62663/commits/e3b049796f2ef52635168d775e967d2cd7cb338f